test/index.js

```diff
@@ -10,76 +10,61 @@ var caches = {
 };
 
 describe('cache control middleware', function() {
-  
+
   var app;
-  
+
   beforeEach(function () {
-    
+
     app = connect()
-      .use(cacheControl(caches));
+      .use(cacheControl(caches))
+      .use(function (req, res, next) {
+        res.statusCode = 200;
+        res.end('OK!');
+      });
   });
-  
+
   it('sets the max age cache header if specified in config file', function (done) {
-    
+
     request(app)
       .get('/index.html')
       .expect('Cache-Control', 'public, max-age=31536000')
       .end(done);
   });
-  
+
   it('sets cache control to no-cache if false is specified in config file', function (done) {
-    
+
     request(app)
       .get('/none.html')
       .expect('Cache-Control', 'no-cache')
       .end(done);
   });
-  
+
   it('sets cache control to the passed string if specified in config file', function (done) {
-    
+
     request(app)
       .get('/private.html')
       .expect('Cache-Control', 'private, max-age=300')
       .end(done);
   });
-  
-  it('sets cache control to 5 minutes? by default', function(done) {
```

Comment:
Hmm, so if we don't set default cache-control header, what we should check here in the test? 🤔